lib/puppet/type/postgresql_replication_slot.rb

```diff
@@ -0,0 +1,9 @@
+Puppet::Type.newtype(:postgresql_replication_slot) do
+  ensurable
```

Comment:
Could you add an `@doc` description, similar to https://github.com/puppetlabs/puppetlabs-mysql/blob/master/lib/puppet/type/mysql_user.rb#L3 (well, more verbose than that would be great).